pygmt/tests/test_datasets_earth_mask.py

```diff
@@ -52,6 +53,7 @@ def test_earth_mask_01d_with_region():
     data = load_earth_mask(resolution="01d", region=[-7, 4, 13, 19])
     assert data.shape == (7, 12)
     assert data.gmt.registration == 0
+    assert data.gmt.gtype == 1
```

Comment:
Check the dtype here too.
```suggestion
    assert data.gmt.gtype == 1
    assert data.dtype == "int8"
```